Misc/NEWS.d/next/C API/2018-11-08-10-54-25.bpo-35081.QVLDPr.rst

```diff
@@ -0,0 +1,3 @@
+Fixed issue where including ``datetime.h`` with the ``PY_BUILD_CORE`` macro
+defined would not expose the C API capsule. ``PY_BUILD_CORE`` is no longer used
+in ``datetime.h``. Patch by Victor Stinner.
```

Comment:
The change fixes an obscure corner case only affecting CPython itself, since Py_BUILD_CORE must not be used outside the CPython code base. I'm not sure that the change deserves a NEWS entry. I suggest to remove it.